libcloud/compute/drivers/gce.py

```diff
@@ -934,10 +935,11 @@ def ex_add_access_config(self, node, name, nat_ip=None, config_type=None):
 
         if nat_ip is not None:
             config['natIP'] = nat_ip
-
+        params = {'accessConfig': name, 'networkInterface': nic}
```

Comment:
@pmoosh - Looks like params should *only* contain 'networkInterface' [https://cloud.google.com/compute/docs/reference/latest/instances/addAccessConfig].  The rest of the data goes into the body of the request (the `config` dict).  I also don't see an explicit reference to attribute/param called `accessConfig`.  Looks like the body should be,

```
{
  "kind": "compute#accessConfig",
  "type": string,
  "name": string,
  "natIP": string
}
```

And we don't need to set `kind`.